ingestion_tools/dataset_configs/deposition_10303.yaml

```diff
@@ -1,37 +1,88 @@
 annotations:
   - metadata:
-      annotation_object:
+      annotation_object: &annotated_object
         id: GO:0016020
         name: Membrane
-        description: Membrain prediction
+        description: Membrain-seg prediction
         state: ~
       dates: &repo-dates
         deposition_date: 2024-05-20
         last_modified_date: 2024-05-20
         release_date: 2024-05-20
-      annotation_method: "membrain"
-      method_type: manual
+      annotation_method: &annotation_method "Prediction using membrain-seg without rescaling and weights MemBrain_seg_v10_alpha.ckpt"
+      method_type: automated
       annotation_publications: ~
       ground_truth_status: False
       authors: &annotation_authors
-        - name: Utz Ermel
+        - name: Utz Heinrich Ermel
+          ORCID: 0000-0003-4685-037X
           primary_annotator_status: true
           corresponding_author_status: true
```

Comment:
is corresponding author correct?

Reply:
Good point, let me ask Anchi and Bridget if they want to be on here.

Reply:
Good for now.